frontend/src/metabase/home/components/HomePage/HomePage.tsx

```diff
@@ -28,6 +33,9 @@ export const HomePage = (): JSX.Element => {
   const isLoading = databaseListState.isLoading || modelListState.isLoading;
   const error = databaseListState.error ?? modelListState.error;
   const dashboardId = useSelector(getCustomHomePageDashboardId);
+  const showHomepageRedirectRoast = useSelector(
+    state => !hasUserDismissedToast(state, "dismissed-custom-dashboard-toast"),
```

Comment:
`getSetting(state, "dismissed..."`

Reply:
I was thinking, since this might be an idea that expands, that having specific selectors and actions around toasts might be good since the setting keys can be typed to just toasts